packages/console/src/pages/Connectors/components/ConnectorRow/index.tsx

```diff
@@ -20,9 +22,20 @@ const ConnectorRow = ({ type, connectors, onClickSetup }: Props) => {
   const inUse = useConnectorInUse(
     conditional(type === ConnectorType.Social && connectors[0]?.target)
   );
+  const navigate = useNavigate();
+
+  const handleClickRow = () => {
+    if (!connectors[0]) {
+      onClickSetup?.();
+
+      return;
+    }
+
+    navigate(`/connectors/${connectors[0].id}`);
+  };
 
   return (
-    <tr>
+    <tr className={tableStyles.clickable} onClick={handleClickRow}>
       <td>
         <ConnectorName type={type} connectors={connectors} onClickSetup={onClickSetup} />
```

Comment:
Should remove this onClickSetup from `<td>` as well

Reply:
Oh, maybe not remove, but stop event propagation?

Reply:
Oh, good call, in this case when "setup" button is visible, this row is not supposed to be clickable, updated.